handlers.py

```diff
@@ -82,6 +95,7 @@ def handle(self, bot, update):
 
 
 definitionHandler = DefinitionHandler(definitionRepo)
+listDefinitionsHandler = ListDefinitionsHandler(definitionRepo)
```

Comment:
The functionality here can be added to the definitions handler (separate method).

Reply:
Thanks, refactored